README.md

```diff
@@ -35,6 +37,7 @@ Alternatively, an external `logback.xml` with much more customizations can be pr
 * Navigate to `docker/service`
 * Add environment tag `export TAG=latest`
 * Navigate to `docker`
+* Add environment tags for AMQP `export GP2GP_AMQP_BROKERS=amqp://activemq:5672`
```

Comment:
This should be the default in docker-compose to avoid having to define lots of extra exports